packages/genesys-spark-tokens/data-old/tokens.json

```diff
@@ -8232,6 +8232,12 @@
           }
         }
       }
+    },
+    "cta_group": {
```

Comment:
Just for my own understanding, what is is cta group exactly?

Reply:
@jason-evans-genesys "Call to action group". Basically a wrapper for buttons to provide consistent positioning.